### PR TITLE
Revert to Qt 5.14.2

### LINKED ISSUE #1581: Sylus pen issues 1.8.1
[Stylus pen on Samsung Galaxy tab Active 2 not functioning. Cannot open project, menus, settings etc. with stylus pen]

### Reproduction Steps:

1. [First Step]
2. [Second Step]
3. [Other Steps...]

### Expected behavior:

[Describe expected behavior here]

### Observed behavior:

[Describe observed behavior here]

### Screenshots and GIFs

Screenshots and GIFs which follow reproduction steps to demonstrate the problem

[Please also attach additional files if a specific project/dataset is useful to investigate the problem.]

**QField version [1.8.1]

## Additional information:

* Problem started happening recently, didn't happen in an older version of QField: [Yes]
* Problem can be reliably reproduced, doesn't happen randomly: [Yes]
* Problem happens with all files and projects, not only some files or projects: [Yes]

### PR DESCRIPTION
**Pull request by m-kuhn**
Fixes #1581

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for a533ced3313467ff5d0c8981799db870dc95f785:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1583-28a8f552e2811cc56c106917a6243490ab96d6ee-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1583-28a8f552e2811cc56c106917a6243490ab96d6ee-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1583-28a8f552e2811cc56c106917a6243490ab96d6ee-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1583-28a8f552e2811cc56c106917a6243490ab96d6ee-x86.apk)

**Comment by m-kuhn** [comment]
https://github.com/opengisch/OSGeo4A/releases/tag/20210108

File: sdk.conf [hunk 1]
```diff
@@ -1 +1 @@
-osgeo4a_version=20210106
+osgeo4a_version=qtrevert
```
Comment (m-kuhn) [inline sdk.conf:1]:
```suggestion
osgeo4a_version=20210108
```

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 2daf9807470b54491aaf689a27bfae9bcb2ad23a:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1583-c7a7289203cb2ade8c57e7f1bffd3c7deab44d5b-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1583-c7a7289203cb2ade8c57e7f1bffd3c7deab44d5b-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1583-c7a7289203cb2ade8c57e7f1bffd3c7deab44d5b-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1583-c7a7289203cb2ade8c57e7f1bffd3c7deab44d5b-x86.apk)

**Comment by nyalldawson** [comment]
... further proof that QtC's recent decisions are absolutely boneheaded...

Enjoy porting this to a broken incomplete qt 6.0 release 😂

**Comment by m-kuhn** [comment]
Or wait for Qt 7 🚀

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 07da6f836fe744337f3230b3ef0b1e2d7afbd7fe:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1583-dc86ed5caf84dffbb2f661ba41d6b0b8f0c5c6e4-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1583-dc86ed5caf84dffbb2f661ba41d6b0b8f0c5c6e4-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1583-dc86ed5caf84dffbb2f661ba41d6b0b8f0c5c6e4-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1583-dc86ed5caf84dffbb2f661ba41d6b0b8f0c5c6e4-x86.apk)

**Comment by m-kuhn** [comment]
@nirvn merge and backport?

**Comment by nirvn** [comment]
Let's do it!

**Review by nirvn — APPROVED** [review]

**Pull request merged**